plugins/crd/plugin_impl_crd.go

```diff
@@ -120,13 +120,13 @@ func networkServiceValidation() *apiextv1beta1.CustomResourceValidation {
 								"name": {
 									Type:        "string",
 									MaxLength:   &maxLength,
-									Description: "NetworkServiceEndpoints Name",
```

Comment:
Pedantic nit: This change is unrelated to the work being done here. I'm fine with cleanups like this as part of this broader merge, but eventually we'd want this type of change as a separate patch.